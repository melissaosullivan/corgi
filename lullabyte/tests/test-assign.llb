void main() {
	int i;
	double d;
	pitch p;
	sound s1;
	sound s2;
	
	i = 1;
	d = 1.0;
	p = A5;
	s1 = |A5|:0.25:100;
	s2 = s1;
	
	print(i);
	print(d);
	print(p);
	print(s1);
	print(s2);
}