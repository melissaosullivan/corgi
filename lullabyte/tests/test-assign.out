1
1.
A5
|A5|:0.25:100
|A5|:0.25:100
